RELEASES.md

```diff
@@ -1,3 +1,641 @@
+- [Add #[must_use] to remaining std functions (A-N)][90430]
+- [Add #[must_use] to remaining std functions (O-Z)][90431]
+- [Windows: Resolve `process::Command` program without using the current directory][87704]
+- [Paths are automatically canonicalized on Windows for operations that support it][89174]
+- [Re-enable `copy[_nonoverlapping]()` debug-checks][90041]
+- [Implement `RefUnwindSafe` for `Rc<T>`][87467]
+- [Make RSplit<T, P>: Clone not require T: Clone][90117]
+- [unix: Make `std::thread::available_concurrency` support process-limited number of CPUs][89310]
+- [Implement `Termination` for `Result<Infallible, E>`][88601]
+
+Stabilized APIs
+---------------
+
+- [`Metadata::is_symlink`][89677]
+- [`Path::is_symlink`][89677]
+- [`{integer}::saturating_div`][88624]
+- [`Option::unwrap_unchecked`][89951]
+- [`NonZero::is_power_of_two`][91301]
+- [`unix::process::ExitStatusExt::core_dumped`][88300]
+- [`unix::process::ExitStatusExt::stopped_signal`][88300]
+- [`unix::process::ExitStatusExt::continued`][88300]
+- [`unix::process::ExitStatusExt::into_raw`][88300]
+
+These APIs are now usable in const contexts:
+
+- [Partially stabilize `duration_consts_2`][89542]
+- [`Duration::new`][89542]
+- [`Duration::checked_add`][89542]
+- [`Duration::saturating_add`][89542]
+- [`Duration::checked_sub`][89542]
+- [`Duration::saturating_sub`][89542]
+- [`Duration::checked_mul`][89542]
+- [`Duration::saturating_mul`][89542]
+- [`Duration::checked_div`][89542]
+- [`Duration::as_secs_f64`][89542]
+- [`Duration::as_secs_f32`][89542]
+- [`Duration::from_secs_f64`][89542]
+- [`Duration::from_secs_f32`][89542]
+- [`Duration::mul_f64`][89542]
+- [`Duration::mul_f32`][89542]
+- [`Duration::div_f64`][89542]
+- [`Duration::div_f32`][89542]
+- [`Duration::div_duration_f64`][89542]
+- [`Duration::div_duration_f32`][89542]
+- [`MaybeUninit::as_ptr`][90896]
+- [`MaybeUninit::as_mut_ptr`][90896]
+- [`MaybeUninit::assume_init`][90896]
+- [`MaybeUninit::assume_init_ref`][90896]
+
+Cargo
+-----
+
+- [Add --message-format for install command][cargo/10107]
+- [Warn when alias shadows external subcommand][cargo/10082]
+
+Misc
+----
+
+- [Show all Deref implementations recursively in rustdoc][90183]
+- [Use computed visibility in rustdoc][88447]
+
+Compatibility Notes
+-------------------
+
+- [All proc-macro backward-compatibility lints are now deny-by-default.][88041]
+- [proc_macro: Append .0 to unsuffixed float if it would otherwise become int token][90297]
+- [Refactor weak symbols in std::sys::unix][90846]
+- `RUSTFLAGS` is no longer set for build scripts. This change was made in
+  1.55, but the release notes did not highlight this change. Build scripts
+  should use `CARGO_ENCODED_RUSTFLAGS` instead. See the
+  [documentation](https://doc.rust-lang.org/nightly/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts)
+  for more details.
+- [rustdoc doctest: detect `fn main` after an unexpected semicolon][91026]
```

Comment:
Maybe mention https://github.com/rust-lang/rust/issues/91398 ? That's the main reason this was marked relnotes.

Reply:
The bug is linked to from that PR, and I can't find examples of us linking to issues for other bug fixes, so I'd rather leave it as is.

Reply:
Rewording for clarity, though.